api/routes/outlier_detecting.go

```diff
@@ -40,22 +46,128 @@ func OutlierDetectionHandler(metaCtor api.MetadataStorageCtor) func(http.Respons
 			return
 		}
 
+		// get the metadata
 		datasetMeta, err := metaStorage.FetchDataset(dataset, false, false, false)
 		if err != nil {
 			handleError(w, err)
 			return
 		}
 
+		// find the outliers in the dataset
 		outlierData, err := task.OutlierDetection(datasetMeta, variable)
 		if err != nil {
 			handleError(w, err)
 			return
 		}
 
+		// create a result
+		result := OutlierResult{
+			OutlierField: "false",
```

Comment:
I haven't looked at the client, but does it check if the variable is named `false` to determine if it worked? It may be better to have a second field in the response that is boolean to determine if the call succeeded or not.